.generated-sources/emily/client/rust/private/docs/Limits.md

```diff
@@ -7,7 +7,9 @@ Name | Type | Description | Notes
 **account_caps** | [**std::collections::HashMap<String, models::AccountLimits>**](AccountLimits.md) | Represents the individual limits for requests coming from different accounts. | 
 **peg_cap** | Option<**u64**> | Represents the total cap for all pegged-in BTC/sBTC. | [optional]
 **per_deposit_cap** | Option<**u64**> | Per deposit cap. If none then there is no cap. | [optional]
+**per_deposit_minimum** | Option<**u64**> | Per deposit minimum. If none then there is no minimum. | [optional]
 **per_withdrawal_cap** | Option<**u64**> | Per withdrawal cap. If none then there is no cap. | [optional]
+**per_withdrawal_minimum** | Option<**u64**> | Per withdrawal minimum. If none then there is no minimum. | [optional]
```

Comment:
Why add withdrawal minimums?

Reply:
I just added all around minimums - we can delete it later if we decide we don't want the option.

Reply:
As a rule, I would keep scope to the minimum.

Reply:
You would keep withdrawal minimum in scope, or you would remove the withdrawal minimum because it's not in scope? @Jiloc and I aren't sure which it is

Reply:
Done.